brainstormforce/astra
# fix: header hidden issue  inside the customizer mobile context

### Description
- Inside customizer mobile context, when previewer refresh happens client width not calculation properly.

### Screenshots
- NA

### Types of changes
- Added a new function to calculate the width regardless of iframe, customizer, or window page.
- Updated the conditions accordingly.

### How has this been tested?
- Check inside customizer mobile / tablet / desktop context.
- Check on the front end.
- Open multiple pages once in chrome.
- Try to break the header.

### Checklist:
